src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/designforextension/InputDesignForExtensionsLocalAnnotations.java

```diff
@@ -0,0 +1,12 @@
+package com.puppycrawl.tools.checkstyle.checks.design.designforextension;
+
+public class InputDesignForExtensionsLocalAnnotations
+{
+    public @interface Rule {
```

Comment:
@romani Like I asked in a similar PR, should we keep this file just for its public interfaces used in another input, or should we merge it so our inputs are completely standalone?